### ISSUE TITLE
Stub files don't parse annotations with parameters

### ISSUE THREAD

**Issue by GoogleCodeExporter** [typetools/checker-framework#97]
```
Niko Matsakis reports a problem with parsing annotations with parameters in 
stub files:

https://groups.google.com/group/checker-framework-discuss/browse_thread/thread/5
94b8c9986fd37fa

The mail contains a patch, but no test cases.

Thanks, Niko!
```

Original issue reported on code.google.com by `wdi...@gmail.com` on 1 Jun 2011 at 11:03
